Makefile

```
@@ -85,6 +85,18 @@ test-heavy: build $(SAMPLEDATA)
 test-pcapingest: bin/$(ZEEKPATH)
 	@ZEEK="$(CURDIR)/bin/$(ZEEKPATH)/zeekrunner" go test -v -run=PcapPost -tags=pcapingest ./ppl/zqd
 
+.PHONY: test-postgres
+test-postgres: build 
+	@ZTEST_PATH="$(CURDIR)/dist:$(CURDIR)/bin" \
+		ZTEST_TAG=postgres \
+		go test -v -run TestZq/ppl/zqd/postgres/ztests .
+
+.PHONY: test-postgres-docker
+test-postgres-docker:
+	@docker-compose -f $(CURDIR)/ppl/zqd/scripts/dkc-services.yaml up -d
+	@-$(MAKE) test-postgres PG_TEST="postgres://test:test@localhost:5432/test?sslmode=disable"
+	@docker-compose -f $(CURDIR)/ppl/zqd/scripts/dkc-services.yaml down
```

Using `-` means this rule can succeed even if the tests fail. Gross as it is, I think this should be something that will fail if either command fails, like this:
```suggestion
	 $(MAKE) test-postgres; \
		status=$?; \
		docker-compose -f $(CURDIR)/ppl/zqd/scripts/dkc-services.yaml down || exit; \
		exit $status
```